Makefile

```diff
@@ -17,8 +17,7 @@ autopep8:
 	autopep8 --recursive --in-place .
 
 pylint:
-	mkdir -p reports
-	PYLINTHOME=reports/ pylint release/
+	pylint release/
```

Comment:
Not related to this PR, But why using pylint and not flake8 as we use in test-infra?

Reply:
We do both